lib/resources/r4/appointment_patch.yaml

```diff
@@ -19,3 +19,77 @@ operations:
       <ul>
         <li>Only "arrived", "checked-in", and "cancelled" are supported</li>
       </ul>
+  - name: add-provider-vmr
+    path: /contained/0/telecom/0/value
+    operation: add
+    cardinality: 0..1
+    type: string
+    url: todo
+    description: The provider video visit link
+    example: |
+      {
+        "op": "add",
+        "path": "/contained/0/telecom/0/value",
+        "value":  "http://providerlink.vmr.net"
+      }
+    note: >
+      <ul>
+        <li>This patch operation may be used only for video visit appointments</li>
+        <li>This patch operation must be accompanied by all other video visit link patch operations</li>
+      </ul>
+  - name: add-patient-vmr
+    path: /contained/1/telecom/0/value
+    operation: add
+    cardinality: 0..1
+    type: string
+    url: todo
+    description: The patient video visit link
+    example: |
+      {
+        "op": "add",
+        "path": "/contained/1/telecom/0/value",
+        "value":  "http://patientlink.vmr.net"
+      }
+    note: >
+      <ul>
+        <li>This patch operation may be used only for video visit appointments</li>
+        <li>This patch operation must be accompanied by all other video visit link patch operations</li>
+      </ul>
+  - name: add-period-start
+    path: /contained/0/telecom/0/period/start
+    operation: add
+    cardinality: 0..1
+    type: datetime
+    url: todo
+    description: The start datetime for the video visit
+    example: |
+      {
+        "op": "add",
+        "path": "/contained/0/telecom/0/period/start",
+        "value":  "2019-07-13T08:00:00.000Z"
+      }
+    note: >
+      <ul>
+        <li>This patch operation may be used only for video visit appointments</li>
+        <li>This patch operation must be accompanied by all other video visit link patch operations</li>
+        <li>The value for this patch operation will be applied to both the provider video visit link and the patient video visit link</li>
+      </ul>
+  - name: add-period-end
+    path: /contained/0/telecom/0/period/end
+    operation: add
+    cardinality: 0..1
+    type: datetime
+    url: todo
```

Comment:
The urls are todo?

Reply:
It looks like that is used for where clicking on the path would take you.
@MaxPhilips do we need to add pages for the contained telecom value and contained telecom period start/end?

Reply:
these links should either be:
* empty
* http://hl7.org/fhir/R4/domainresource-definitions.html#DomainResource.contained

the second is probably better

Reply:
The telecom ones should be:

https://github.com/cerner/fhir.cerner.com/blob/master/lib/resources/r4/patient_patch.yaml#L299

The Period end ones should be:

https://github.com/cerner/fhir.cerner.com/blob/master/lib/resources/r4/patient_patch.yaml#L333

Reply:
So Max says to make the telecom urls:
http://hl7.org/fhir/R4/domainresource-definitions.html#DomainResource.contained
and Evan says:
https://hl7.org/fhir/R4/datatypes-definitions.html#ContactPoint.value

Which is correct?

Reply:
Yeah Im even going to change what we should have for the Period ones. Since they are for the individual start and end times they should be the following links:

http://hl7.org/fhir/R4/datatypes-definitions.html#Period.start
http://hl7.org/fhir/R4/datatypes-definitions.html#Period.end

Reply:
https://github.com/cerner/fhir.cerner.com/pull/294/commits/1a4e99ea8f7c610cc24a6b9d0af3b6557cdb4dd9
https://github.com/cerner/fhir.cerner.com/pull/294/commits/d67d2b50bc330777ca0376a9740a944cb0af57c6

Reply:
+1